src/main/java/dev/latvian/mods/kubejs/command/DumpCommands.java

```diff
@@ -79,9 +78,7 @@ public static int events(CommandSourceStack source) {
 
 				if (eventType.getPackageName().startsWith("dev.latvian.mods.kubejs")) {
 					builder.append('[').append(UtilsJS.toMappedTypeString(eventType)).append(']')
-						.append('(').append("https://github.com/KubeJS-Mods/KubeJS/tree/")
-						.append(KubeJS.MC_VERSION_NUMBER)
-						.append("/common/src/main/java/")
+						.append('(').append("https://github.com/KubeJS-Mods/KubeJS/tree/main/src/main/java/")
```

Comment:
I don't like the use of `main` as it will become outdated once KJS is ported to a new MC version.

Reply:
I can't think of anything better I can do. Do you have any ideas?

Reply:
Oh wait yeah, there is no 1.21 branch, just a `main` branch. Maybe it does just need to use `main`.

Reply:
I'll modify it when I get back to the computer. Don't merge it yet.

Reply:
I think it can be merged?